models/base_sources/arbitrum_base_sources.yml

```diff
@@ -265,31 +265,82 @@ sources:
         loaded_at_field: evt_block_time
         description: "Transfers events for ERC721 tokens on Arbitrum."
         columns:
-          - &contract_address
-            name: contract_address
+          - name: contract_address
             description: "ERC721 token contract address"
           - *evt_tx_hash
           - *evt_index
           - *evt_block_time
           - *evt_block_number
           - *from
           - *to
-          - &tokenId
-            name: tokenId
+          - name: tokenId
             description: "ERC721 token ID."
 
+      - name: evt_ApprovalForAll
```

Comment:
this source file seems broken -- `evt_transferbatch` is listed as schema based on indentation & usage of tables within. i think it needs to be a table, within another schema? that bug has led to incorrect usage for new `evt_ApprovalForAll` as well

Reply:
quick glance at the avalanche source file shows this is truly a bug, it looks correct over there. can you go back through each file and ensure they are setup properly in this regard of schemas/tables layout?

Reply:
yeah I will work on a fix, thanks for flagging!